modulefiles/module_base.hera.lua

```diff
@@ -41,9 +41,17 @@ load(pathJoin("w3nco", "2.4.1"))
 load(pathJoin("wgrib2", "2.0.8"))
 setenv("WGRIB2","wgrib2")
 
-append_path("MODULEPATH", "/contrib/anaconda/modulefiles")
-load(pathJoin("anaconda", "2.3.0"))
-
 load(pathJoin("cdo", "1.9.5"))
 
+load(pathJoin("R", "3.5.0"))
+
+-- Temporary until official hpc-stack is updated
+prepend_path("MODULEPATH", "/scratch2/NCEPDEV/ensemble/save/Walter.Kolczynski/hpc-stack/modulefiles/stack")
+load(pathJoin("hpc", "1.2.0"))
```

Comment:
Will the official hpc-stack you're waiting for be `1.2.0` or will it stay `1.1.0`?

Reply:
This is only for the temporary custom portion.